gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcUtils.java

```diff
@@ -75,10 +75,10 @@ private OrcUtils() {
 
   public static TypeDescription getTypeDescriptionFromFile(Configuration conf, Path orcFilePath)
       throws IOException {
-    return getRecordReaderFromFile(conf, orcFilePath).getSchema();
+    return getFileReader(conf, orcFilePath).getSchema();
   }
 
-  public static Reader getRecordReaderFromFile(Configuration conf, Path orcFilePath)
+  public static Reader getFileReader(Configuration conf, Path orcFilePath)
```

Comment:
Since this is a public static method, are we sure there is no other dependencies call this API?

Reply:
Will put an umbrella method for backward compatibility.